### ISSUE TITLE
connectors are not flush

### ISSUE THREAD

**Issue by sbourdeauducq** [sinara-hw/SMA_IDC_Adapter#2]
With the straight SMA connectors assembled against the PCB (as seems to be the case on the TS boards), they become slightly shorter than the 90-degree SMA connectors.
Was this done so that there is enough space between the connector body and the front panel? It seems (at least with the Schroff panels) that the connector could be advanced a little.

**Comment by gkasprow** [comment]
It looks like they used some replacement parts. On my 3D model, they fit precisely.
AFAIR I copied them with the layout from DIO-SMA
Can you paste some photos?

**Comment by HarryMakes** [comment]
Hello Greg, Sebastien and I saw this issue from the board shown below. The black markings on the paper (top of the pictures) underneath were to show the approximate height difference between the two types of SMA connectors, which was measured to be ~0.6mm. Thank you!

Straight SMA heads (shorter) | 90-deg SMA heads (taller)
-|-
![20210128-IDCSMA-flushing-01](https://user-images.githubusercontent.com/66358631/106078415-579a3780-614e-11eb-802b-ecda51d8a730.jpg) | ![20210128-IDCSMA-flushing-02](https://user-images.githubusercontent.com/66358631/106078425-59fc9180-614e-11eb-8746-b84e35df4383.jpg)

**Comment by HarryMakes** [comment]
On this TS board, the inside of the straight SMA connectors are also touching the edge of the PCB as shown below.

![20210128-IDCSMA-flushing-03](https://user-images.githubusercontent.com/66358631/106080625-6be03380-6152-11eb-965e-89f7414eb02e.jpg)

**Comment by pkozakiewicz** [comment]
> It looks like they used some replacement parts. On my 3D model, they fit precisely.

No, we always use Adam-Tech RF249BT0050GHDW as listed in BOM. It seems that the layout needs some amendments.

**Comment by gkasprow** [comment]
Is it the same issue with DIO-SMA?
Don't use SMA nuts because there won't be enough thread for proper SMA connector fitting

**Comment by pkozakiewicz** [comment]
> Is it the same issue with DIO-SMA?
> Don't use SMA nuts because there won't be enough thread for proper SMA connector fitting

Yes, DIO-SMA is facing similar problems but there is >4.8mm thread left anyway.
We started to do this on the special request of some of our customers, we then use thick heatshrinks to seal it.
